### PR TITLE
Add WPF V141 Photon Fractal detector 

### PR DESCRIPTION
**Pull request by xuzhenpeng263**
### Method                                                                    
  - **Features**: 60-dim language-independent statistical features                     
  - **Classifier**: XGBoost                                                     
  - **Training**: RAID-train (all domains, including adversarial attacks),      
  per-domain balanced sampling (Human:AI = 1:1)                                 
  - **Languages**: Multilingual (no language-specific features)                 
                                                                                
  ### Key Characteristics                                                       
  - Pure statistical features, no neural network or pre-trained language model  
  - No word lists or lexicons required                                          
  - Supports all languages in RAID (English, Czech, German, Code)

### CODE REVIEW

**Comment by liamdugan** [comment]
Congrats on the strong performance @xuzhenpeng263 Let me know if you'd like to merge the results

**Comment by xuzhenpeng263** [comment]
> Congrats on the strong performance @xuzhenpeng263 Let me know if you'd like to merge the results

thanks，I need

**Comment by xuzhenpeng263** [comment]
> Congrats on the strong performance @xuzhenpeng263 Let me know if you'd like to merge the results

Sure, I’d like to merge the results.
